src/sandmastery/Java/leaf/cosmere/sandmastery/common/blocks/entities/TemporarySandBE.java

```diff
@@ -0,0 +1,45 @@
+package leaf.cosmere.sandmastery.common.blocks.entities;
+
+import leaf.cosmere.sandmastery.common.registries.SandmasteryBlockEntitiesRegistry;
+import net.minecraft.core.BlockPos;
+import net.minecraft.nbt.CompoundTag;
+import net.minecraft.world.level.Level;
+import net.minecraft.world.level.block.Blocks;
+import net.minecraft.world.level.block.entity.BlockEntity;
+import net.minecraft.world.level.block.state.BlockState;
+
+import java.util.concurrent.ThreadLocalRandom;
+
+import static leaf.cosmere.sandmastery.common.blocks.TemporarySandBlock.AGE;
+
+public class TemporarySandBE extends BlockEntity {
```

Comment:
I'd also like { to be on their own line, just to stay consistent with the rest of the project

Reply:
Other files include:

- SandmasteryKeybindings
- SandJarBE
- SandSpreaderBE
- TemporarySandBE
- TaldainSandBlock
- TaldainSandLayerBlock
- TemporarySandBlock
- MasteryPlatform
- SandmasteryManifestations
- sandmastery.common.utils.MiscHelper
-